big-o-notation/big-o.js

```diff
@@ -0,0 +1,44 @@
+/* exported uniqueQuadratic, uniqueLinear */
+
+function uniqueLinear(words) {
+  const seen = {};                  // 1 * 1 = O(1)
+  const unique = [];                // 1 * 1 = O(1)
+  for (
+    let i = 0;                      // 1 * 1 = O(1)
+    i < words.length;               // 3 * n = O(3n) ~= O(n)
+    i++                             // 3 * n = O(3n) ~= O(n)
+  ) {
+    const word = words[i];          // 3 * n = O(3n) ~= O(n)
+    if (!seen[word]) {              // 2 * n = O(2n) ~= O(n)
+      seen[word] = true;
+      unique[unique.length] = word; // 3 * 3 = O(3n)
```

Comment:
This would be 2 operations, access `length` property `1`, and assign `1`. Assigning to a specific property only counts as 1  operation.